File: EJERCICIO_1/EJERCICIOJAVA/holiDay.java
package EJERCICIOJAVA;

public class holiDay extends bonus {
    public holiDay(salaryBass salary, double  porcentage , int days) {
        super(salary, 2.00, days); 
    }

    public double calculateBonus() {
        return valueDay * percentage * days; 
    }
}

File: EJERCICIO_1/EJERCICIOSPHP/holiDay.php
<?php

class Holiday extends Bonus {
    public function __construct(SalaryBass $salaryBass, $percentage, $days) {
        parent::__construct($salaryBass, 2.00, $days); // Fixed percentage of 2.00 for holidays
    }

    public function calculateBonus() {
        return $this->valueDay * $this->percentage * $this->days; // Calculate the bonus
    }
}

?>